File: 1037.cpp
#include <stdio.h>
int main()
{
    double a;
    scanf ("%lf",&a);
    if(a<0||a>100)
        printf ("Fora de intervalo\n");
    else if(a>=0&&a<=25)
        printf ("Intervalo [0,25]\n");
    else if(a>25&&a<=50)
        printf ("Intervalo (25,50]\n");
    else if (a>50&&a<=75)
        printf ("Intervalo (50,75]\n");
    else if (a>75&&a<=100)
        printf ("Intervalo (75,100]\n");
    return 0;
}
//Vinicius Mangueira Correia

File: Answers/Python/1037.py
# -*- coding: utf-8 -*-

n = float(raw_input())

if n > 100.0 or n < 0.0:
	print "Fora de intervalo"
else:
	if 0.0 <= n <=25.0:
		print "Intervalo [0,25]"
	
	if 25.0 < n <=50.0:
		print "Intervalo (25,50]"
	
	if 50.0 < n <=75.0:
		print "Intervalo (50,75]"
	
	if 75.0 < n <=100.0:
		print "Intervalo (75,100]"
